### PR TITLE
badges: ui implementation for badges in zenodo

### PR DESCRIPTION
**Pull request by TLGINO**
" added suport for:
  - F1000
  - Dryad
  - Github
  - OpenAire
  - Orcid
  - Reana
  - Inspire
 * closes https://github.com/zenodo/zenodo-rdm/issues/249

### CODE REVIEW

File: site/zenodo_rdm/utils.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Utility functions."""
+
+
+def github_link_render(record):
+    """Entry for Github."""
+    # [TODO] Integrate Inveio_Github here
+    resources = []
+
+    related_identifiers = record["ui"].get("related_identifiers", [])
```
Comment (anikachurilova) [inline site/zenodo_rdm/utils.py:15]: Little (maybe stupid) comment/quiestion: do we know for sure that `record["ui"]` exists?
Is it better to use `lodash/get` here instead?

Comment (TLGINO) [reply site/zenodo_rdm/utils.py:15]: from tests and debugging it looks to me like `ui` always exists although this doesn't seem to be the case for `related_identifiers` (hence the `get`).
But you're right using `get` for both is probably safer

File: site/zenodo_rdm/utils.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    return resources or None
+
+
+def openaire_link_render(record):
+    """Entry for Openaire."""
+    resources = []
+
+    related_identifiers = record.get("ui", {}).get("related_identifiers", [])
+    for identifier in related_identifiers:
+        if identifier["scheme"] == "url" and "openaire" in identifier["identifier"]:
```
Comment (kpsherva) [inline site/zenodo_rdm/utils.py:35]: missing top level domain for openaire? ping @slint

**Comment by slint** [comment]
Screenshot:

image

We're missing an icon for the Journal of Brief Ideas, but can be done in a separate issue.

**Pull request merged**